### PR TITLE
[DOCS] Clarify what owning contract means

### LINKED ISSUE #3395: What is an owning contact?
While going through *solidity* [documentation](https://solidity.readthedocs.io/en/develop/introduction-to-smart-contracts.html) I came across a term 'the owning contract' which i didnt understand.

> A contract in the sense of Solidity is a collection of code (its functions) and data (its state) that resides at a specific address on the Ethereum blockchain. The line uint storedData; declares a state variable called storedData of type uint (unsigned integer of 256 bits). You can think of it as a single slot in a database that can be queried and altered by calling functions of the code that manages the database. In the case of Ethereum, this is always **the owning contract**. And in this case, the functions set and get can be used to modify or retrieve the value of the variable.

### PR DESCRIPTION
**Pull request by ChrisChinchilla**
### Description

This PR closes https://github.com/ethereum/solidity/issues/3395 as i think for an introduction section, maybe it's not clear enough what "owning" means.

### Checklist
- [x] Code compiles correctly
- [x] All tests are passing
- [x] New tests have been created which fail without the change (if possible)
- [x] README / documentation was extended, if necessary
- [x] Changelog entry (if change is visible to the user)
- [x] Used meaningful commit messages

### CODE REVIEW

File: docs/introduction-to-smart-contracts.rst [hunk 1]
```diff
@@ -43,7 +43,7 @@ blockchain. The line ``uint storedData;`` declares a state variable called ``sto
 type ``uint`` (*u*\nsigned *int*\eger of *256* bits). You can think of it as a single slot
 in a database that you can query and alter by calling functions of the
 code that manages the database. In the case of Ethereum, this is always the owning
```
Comment (chriseth) [inline docs/introduction-to-smart-contracts.rst]: Maybe it is easier to understand if we just remove the sentence 'In the case of Ethereum, this is always the owning contract.`?

Comment (ChrisChinchilla) [reply docs/introduction-to-smart-contracts.rst]: Personally I feel it should be left in. This is the introduction, maybe no one knows what Ethereum is yet. We make a comparison above to a database, which many developers understand, but then we say what the equivalent of that is with Ethereum.

Comment (chriseth) [reply docs/introduction-to-smart-contracts.rst]: I agree, but the paragraph does the comparison. Then we explain which functions in ethereum can modify the value.

Comment (ChrisChinchilla) [reply docs/introduction-to-smart-contracts.rst]: OK, removed, but added an additional few words after.

**Review by chriseth — APPROVED** [review]

**Pull request merged**